bot/exts/info/subscribe.py

```diff
@@ -181,18 +186,12 @@ async def subscribe_command(self, ctx: commands.Context, *_) -> None:  # We don'
             row = index // ITEMS_PER_ROW
             button_view.add_item(SingleRoleButton(role, role.role_id in author_roles, row))
 
-        await ctx.reply(
+        await ctx.send(
```

Comment:
this can use the reference, with a fail_if_not_exists IIRC

Reply:
I see this exists https://discordpy.readthedocs.io/en/master/api.html#discord.Message.to_reference but that's for creating message references. So it could be used by adding `reference=ctx.message.to_reference(fail_if_not_exists=False),` to the ctx.send call.

However fail_if_not_exists is just for messages that are deleted/NotFound before the send could happen. The API still raises the error `discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body In message_reference: Cannot reply to a message in a different channel` if the message exists, but is in another channel.

Reply:
Huh, that's interesting, and a bit annoying, I guess.

Reply:
> 

Agreed :D